extensions/amp-story-auto-ads/0.1/test-e2e/test-amp-story-auto-ads.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const viewport = {
+  HEIGHT: 823,
+  WIDTH: 500,
+};
+
+describes.endtoend(
+  'amp-story-auto-ads',
+  {
+    testUrl:
+      'http://localhost:8000/test/manual/amp-story-auto-ads/amp-story-auto-ads-basic.html',
```

Comment:
is test/manual the right place to put e2e fixtures?

Reply:
`test/fixtures/e2e/amp-story-auto-ads/basic.html` would probably be better

Reply:
Moved.